diff --git a/.gitignore b/.gitignore
index bba0ec2..5dbf77b 100644
--- a/.gitignore
+++ b/.gitignore
@@ -3,11 +3,7 @@
 /.powrc
 /.rvmrc
 /config/additional_environment.rb
-/config/configuration.yml
 /config/database.yml
-/config/email.yml
-/config/initializers/session_store.rb
-/config/initializers/secret_token.rb
 /coverage
 /db/*.db
 /db/*.sqlite3
@@ -20,7 +16,6 @@
 /plugins/*
 !/plugins/README
 /public/dispatch.*
-/public/plugin_assets/*
 /public/themes/*
 !/public/themes/alternate
 !/public/themes/classic
@@ -37,6 +32,4 @@
 *.rbc
 
 /.bundle
-/Gemfile.lock
-/Gemfile.local
 
