diff --git a/.gitignore b/.gitignore
index 5e0f888..1610377 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,11 +1,7 @@
 /.project
 /.loadpath
 /config/additional_environment.rb
-/config/configuration.yml
 /config/database.yml
-/config/email.yml
-/config/initializers/session_store.rb
-/config/initializers/secret_token.rb
 /coverage
 /db/*.db
 /db/*.sqlite3
@@ -16,7 +12,6 @@
 /log/*.log*
 /log/mongrel_debug
 /public/dispatch.*
-/public/plugin_assets
 /tmp/*
 /tmp/cache/*
 /tmp/pdf/*
@@ -29,6 +24,4 @@
 *.rbc
 
 /.bundle
-/Gemfile.lock
-/Gemfile.local
 
